CHANGELOG-developer.next.asciidoc

```diff
@@ -92,3 +92,5 @@ The list below covers the major changes between 7.0.0-rc2 and master only.
 - Added SQL helper that can be used from any Metricbeat module {pull}18955[18955]
 - Update Go version to 1.14.4. {pull}19753[19753]
 - Update Go version to 1.14.7. {pull}20508[20508]
+- Add packaging for docker image based on UBI minimal 8. {pull}20576[20576]
```

Comment:
extra line

Reply:
can't wait to not have to deal with changelog.